lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -54,56 +54,13 @@ defmodule Kernel.SpecialForms do
   @doc """
   Creates a map.
 
-  Maps are key-value stores where keys are compared
-  using the match operator (`===`). Maps can be created with
-  the `%{}` special form where keys are associated via `=>`:
-
-      %{1 => 2}
-
-  Maps also support the keyword notation, as other special forms,
-  as long as they are at the end of the argument list:
-
-      %{hello: :world, with: :keywords}
-      %{:hello => :world, with: :keywords}
-
-  If a map has duplicated keys, the last key will always have
-  higher precedence:
-
-      iex> %{a: :b, a: :c}
-      %{a: :c}
-
-  Conveniences for manipulating maps can be found in the
-  `Map` module.
-
-  ## Access syntax
-
-  Besides the access functions available in the `Map` module,
-  like `Map.get/3` and `Map.fetch/2`, a map can be accessed using the
-  `.` operator:
-
-      iex> map = %{a: :b}
-      iex> map.a
-      :b
-
-  Note that the `.` operator expects the key `:a` to exist in the map.
-  If not, an `ArgumentError` is raised.
-
-  ## Update syntax
-
-  Maps also support an update syntax:
-
-      iex> map = %{:a => :b}
-      iex> %{map | :a => :c}
-      %{:a => :c}
-
-  Notice the update syntax requires the given keys to exist.
-  Trying to update a key that does not exist will raise an `KeyError`.
+  See the `Map` module for more information about maps, their syntax, and ways to
+  access and manipulate them.
 
   ## AST representation
 
-  Regardless if `=>` or the keywords syntax is used, Maps are
-  always represented internally as a list of two-element tuples
-  for simplicity:
+  Regardless if `=>` or the keywords syntax is used, key-value pairs in maps are
```

Comment:
Looks good to me. Let's just make sure we are not losing any information in
the migration.

On Tuesday, July 5, 2016, Eric Entin <notifications@github.com> wrote:

> In lib/elixir/lib/kernel/special_forms.ex
> <https://github.com/elixir-lang/elixir/pull/4947#discussion_r69598101>:
>
> >
> >    ## AST representation
> >
> > -  Regardless if `=>` or the keywords syntax is used, Maps are
> > -  always represented internally as a list of two-element tuples
> > -  for simplicity:
> > +  Regardless if `=>` or the keywords syntax is used, key-value pairs in maps are
>
> Maybe "Regardless of whether =>" would be more appropriate? Not super
> important but while this is being changed might as well, right?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4947/files/a715b1cac0f143857f152aa12dabae23ae10f335#r69598101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAlbh4Hk-pYmhDXcwgo2M30BriEIz7tks5qSotUgaJpZM4JFTxb>
> .
>


-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D